Taskfile.yml

```diff
@@ -19,3 +19,18 @@ tasks:
     desc: Runs Golang integration tests
     cmds:
       - go test -tags=integration ./tests/integration/...
+
+  docker-build:
+    desc: Builds Docker image
+    cmds:
+      - docker build -t corbado-go:latest .
+    sources:
+      - ./Dockerfile
+    method: checksum
+
+  generate-openapi-entities:
```

Comment:
since its not only entities anymore use a different name. "generate-openapi" ? "generate-openapi-code" ?